ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts

```diff
@@ -163,7 +163,7 @@ export interface MhRegistrationSummaryIF {
   documentRegistrationNumber?: string
   expand?: boolean // used in UI table to toggle expansion.
   expireDays?: number // Number of days until expiry
-  lienRegistrationType?: string
+  lienRegistrationType?: string,
```

Comment:
Did you mean to add the the comma here?

Reply:
Definitely not in an interface, good catch!

Reply:
Fixed